tests/pactHelper.js

```diff
@@ -260,7 +278,7 @@ function setGeneralInteractions (provider) {
       headers: xmlResponseHeaders,
       body: '<?xml version="1.0"?>\n' +
         '<ocs>\n' +
-        ocsSuccessMeta +
+        ocsMeta('ok', '100') +
         '  <data/>\n' +
         '</ocs>\n'
     }
```

Comment:
IMO covers more duplication if this can be managed like:
```
body: ocsResponseXml('ok', '100')
```

Reply:
that might create confusions later. So i think it is better to do as it is done now, since the code inside the meta part is similar almost everywhere with just different value of status and status code. the other elements such as data and  element are not easy to manage in the same function or variable